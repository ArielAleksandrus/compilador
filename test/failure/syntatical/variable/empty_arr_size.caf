int soma(int a, int b){
	int c[];
}
programa{}