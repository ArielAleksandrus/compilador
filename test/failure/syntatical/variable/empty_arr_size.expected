ERROR => Line 2, col 7: Invalid array size expression
